game/scripts/vscripts/addon_game_mode.lua

```diff
@@ -160,6 +163,39 @@ function CMegaDotaGameMode:OnEntityKilled( event )
     local killedUnit = EntIndexToHScript( event.entindex_killed )
     local killer = EntIndexToHScript( event.entindex_attacker )
 	local killedTeam = killedUnit:GetTeam()
+	local name = killedUnit:GetUnitName()
+
+	local raxbonuses = {
```

Comment:
With `raxBonuses` as a global that name would be confusing
```suggestion
	local raxRespawnTimeWorth = {
```